src/promptflow-evals/pyproject.toml

```diff
@@ -42,6 +42,10 @@ python = "<4.0,>=3.8"
 azure-ai-ml = ">=1.14.0"
```

Comment:
Can we remove it now?

Reply:
removed. we should make sure the get_rai_svc_url works on the evals without this.